### PR TITLE
Set empty dict as `output` for jobs

### PR DESCRIPTION
**Pull request by shtripat**
This would be helpful for flows/atoms for updating the output
data.

Signed-off-by: Shubhendu <shtripat@redhat.com>

### CODE REVIEW

File: tendrl/commons/objects/job/__init__.py [hunk 1]
```diff
@@ -22,6 +22,7 @@ def __init__(self, job_id=None, payload=None,
     def save(self):
         super(Job, self).save()
         payload = json.loads(self.payload)
+        output = json.loads(self.output)
         if "parent" in payload:
```
Comment (r0h4n) [inline tendrl/commons/objects/job/__init__.py]: Not required

File: tendrl/commons/jobs/__init__.py [hunk 2]
```diff
@@ -55,6 +55,11 @@ def _run(self):
                             pass
 
                         job = Job(job_id=jid).load()
+
+                        # Set the empty dict as output for the job
+                        job.output = json.dumps({})
+                        job.save()
```
Comment (r0h4n) [inline tendrl/commons/jobs/__init__.py]: Just dict() , remove json.dumps

File: tendrl/commons/objects/job/__init__.py (hunk 1 above)
Comment (shtripat) [inline tendrl/commons/objects/job/__init__.py]: Done

File: tendrl/commons/jobs/__init__.py (hunk 2 above)
Comment (shtripat) [inline tendrl/commons/jobs/__init__.py]: Just `dict()` doesnt work. It works with `dict(_None=None)`

**Pull request merged**